test/client/fixtures/sandbox/event/active-window-tracker-test.js

```diff
@@ -92,3 +92,43 @@ asyncTest('check switching active window (between two iframes)', function () {
             });
     });
 });
+
+module('regression');
+
+asyncTest('check that error do not rises when try to send serviceMessage to removed iframe (GH-206)', function () {
```

Comment:
check that an error does not rise when trying to send serviceMessage to the removed iframe (GH-206)